packages/twenty-server/src/engine/api/graphql/graphql-query-runner/graphql-query-parsers/graphql-query-selected-fields/graphql-selected-fields.parser.ts

```diff
@@ -83,9 +89,7 @@ export class GraphqlQuerySelectedFieldsParser {
   }
 
   private shouldNotParseField(fieldKey: string): boolean {
-    return ['__typename', 'totalCount', 'pageInfo', 'cursor'].includes(
```

Comment:
Should we replace totalCount everywhere in the app by countUniqueId? We'll need to give a heads up for deprecation but we can start replacing it in our frontend at least. We plan to email people about API Key deprecation so we can let them know about this at the same time. 
If I'm not mistaken we're doing a separate query for totalCount now while this could fit in the same query with all other aggregated fields?